tests/sqlparser_common.rs

```diff
@@ -4026,6 +4026,120 @@ fn parse_alter_table() {
     }
 }
 
+#[test]
+fn parse_rename_table() {
+    let dialects = all_dialects_where(|d| d.supports_rename_table());
+
+    match dialects.verified_stmt("RENAME TABLE `test`.`test1` TO `test_db`.`test2`") {
+        Statement::RenameTable { operations } => {
+            assert_eq!(
+                vec![RenameObjectDef {
+                    old_name: ObjectName(vec![
+                        Ident {
+                            value: "test".to_string(),
+                            quote_style: Some('`'),
+                            span: Span::empty(),
+                        },
```

Comment:
in order to shorten the test case can we replace occurrences of `Ident {...}` with `Ident::new()` and `Ident::with_quote()` where applicable?

Reply:
Good idea! Have Done.